stack.yaml

```diff
@@ -1,74 +1,11 @@
-# This file was automatically generated by 'stack init'
```

Comment:
I can, but note that I did not manually edit the `stack.yaml` file.  What you see in that file is only the result of running `stack solver --update-config`.  So those comments and things can be retained, but then, every time you tell Stack to update the config, you're going to have to manually edit `stack.yaml` to keep them.